pkg/agent/runtime.go

```diff
@@ -58,11 +60,12 @@ func (a *DebugAttacher) AttachContainer(name string, uid kubetype.UID, container
 }
 
 // GetAttacher returns an implementation of Attacher
-func (m *RuntimeManager) GetAttacher(image string, authStr string, command []string, context context.Context, cancel context.CancelFunc) kubeletremote.Attacher {
+func (m *RuntimeManager) GetAttacher(image, authStr, lxcfsEnabled string, command []string, context context.Context, cancel context.CancelFunc) kubeletremote.Attacher {
```

Comment:
I suggest to define the `lxcfsEnabled` parameter as boolean and convert the form value in the caller code